src/Components/Icons.jsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable react/no-unknown-property */
+export const CalendarIcon = ({ size }) => (
+  <svg
+    width={size}
+    height={size}
+    viewBox="0 0 14 14"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
+      d="M1.75 3.50004C1.75 2.85571 2.27233 2.33337 2.91667 2.33337H11.0833C11.7277 2.33337 12.25 2.85571 12.25 3.50004V11.6667C12.25 12.311 11.7277 12.8334 11.0833 12.8334H2.91667C2.27233 12.8334 1.75 12.311 1.75 11.6667V3.50004Z"
+      stroke="#422F8A"
+      stroke-opacity="0.87"
```

Comment:
@9xVibee can you change these from kebab casing to camel casing like strokeOpacity